core/server/translations/en.json

```diff
@@ -441,7 +441,7 @@
             },
             "icons": {
                 "missingFile": "Please select an icon.",
-                "invalidFile": "Blog icon must be a square .ico or .png file between 60px – 1,000px, under 100kb.",
+                "invalidFile": "Icon must be a square .ico or .png file between 60px – 1,000px, under 100kb.",
```

Comment:
Removed "Blog" here as this is as the translation key naming suggests it's supposed to be a generic icon error message. Example in-situ:

<img width="810" alt="screenshot 2019-03-07 at 15 26 08" src="https://user-images.githubusercontent.com/415/53967547-5c364c80-40ed-11e9-8752-beea34ff67f2.png">